### PR TITLE
Initial Contribution of Dataspace Protocol 2024-01

### PR DESCRIPTION
**Pull request by ssteinbuss**
This is the initial contribution of the Dataspace Protocol 2024-01 based on the work of International Data Spaces Association.

### CODE REVIEW

**Comment by arnoweiss** [comment]
So this is "just" a dump of the the IDSA repository [1], right? This will erase the git history prior to this point. I'm not saying that this is a problem but it should be a concious decision.

[1] https://github.com/International-Data-Spaces-Association/ids-specification/tree/main

File: README.md [hunk 1]
```diff
@@ -1,2 +1,79 @@
-# DataspaceProtocol
-The Dataspace Protocol is a set of specifications designed to facilitate interoperable data sharing between entities governed by usage control and based on Web technologies. These specifications define the schemas and protocols required for entities to publish data, negotiate Agreements, and access data in a data space 
+# Dataspace Protocol Release 2024-1
+
+> __NOTE:__ For GitHub users, the link to the rendered content is [https://docs.internationaldataspaces.org/dataspace-protocol/](https://docs.internationaldataspaces.org/dataspace-protocol/).
```
Comment (ssteinbuss) [inline README.md:3]: Should be removed

File: README.md [hunk 2]
```diff
@@ -1,2 +1,79 @@
...
+>
+> __NOTE:__ The human-friendly version of this specification in the [IDSA Knowledge base](https://docs.internationaldataspaces.org/dataspace-protocol/) will always show the latest version of the document. The version history and changes are provided via the [GitHub Repository](https://github.com/International-Data-Spaces-Association/ids-specification/).
```
Comment (ssteinbuss) [inline README.md:5]: Shold be removed

File: model/model.md
Comment (anilturkmayali) [inline model/model.md:1]: I think the title Dataspace Model could lead to some confusion. Using "Dataspace Information Model" could be better. 
Fyi, the link heading here is titled as "Information Model".

I think for all parts in the repo we need to have a unique term and "Dataspace Information Model" or "Information Model for Data Spaces" could be better options.

File: best.practices/extensions.md
Comment (anilturkmayali) [inline best.practices/extensions.md:1]: If this is under development, would be better to write it like that. At least some people would be interested in extending the DSP :D

**Review by anilturkmayali — APPROVED** [review]
Finished. Just added a couple of comments that we can handle after the merge.

**Pull request merged**